src/Compiler/Interactive/fsi.fs

```diff
@@ -1646,6 +1646,32 @@ type internal FsiDynamicCompiler(
         diagnosticsLogger.AbortOnError(fsiConsoleOutput)
         codegenResults, optEnv, fragName
 
+    /// Check FSI entries for the presence of EntryPointAttribute and issue a warning if it's found
+    let CheckEntryPoint (declaredImpls: CheckedImplFile list) =
```

Comment:
```suggestion
    let WarnIfEntryPoint (declaredImpls: CheckedImplFile list) =
```

Reply:
I think I like `Check` better. It's kind of up to that function to decide what to do if it finds it. There was also a discussion about raising an error instead of warning. But if there are any more votes for `Warn` I'll rename it :)

Reply:
I'd expect Check only controlling if things are fine without acting on that. But that's not a big deal, feel free to resolve :)